libraries/botbuilder-ai/src/luisRecognizer.ts

```diff
@@ -205,6 +210,35 @@ export class LuisRecognizer {
         });
     }
 
+    private prepareErrorMessage(error: Error): Error {
```

Comment:
could error be null / undefined?

Reply:
Technically where it's called it could never be `null` or `undefined` unless someone changed the catch  logic to change `error`.